### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by splattner**
This container image is for the script mentioned in https://github.com/appuio/rancher-docs/pull/3
It contains the az tools with the needed extentsion, all required other tools and the update update script

### CODE REVIEW

**Comment by splattner** [comment]
For the actions, I guess the env with credentials are yet missing. I cannot add them by myself. Base for this is https://github.com/appuio/container-helm
so if someone could configure these envs for me, that would be great!

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2018, VSHN AG, info@vshn.ch
```
Comment (anothertobi) [inline LICENCE]: The year is a bit outdated.

```suggestion
Copyright (c) 2021, VSHN AG, info@vshn.ch
```

File: 2.9.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (anothertobi) [inline 2.9.1/Dockerfile]: I think it's OK to start on the first line

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (anothertobi) [inline .github/workflows/pr.yaml]: First line can be used

File: README [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+> [APPUiO](https://appuio.ch) -
+> GitHub [@appuio](https://github.com/appuio) -
+> Twitter [@appuio](https://twitter.com/appuio)
+
+## Usage
+
+### Nodepool image upgrade
+
+```bash
+docker run -it appuio/aks-maintenance: updateNodes.sh <resource-group> <cluster-name> <nodepool-name> [apply]
```
Comment (anothertobi) [inline README]: I would add the domain to this command:
```suggestion
docker run -it docker.io/appuio/aks-maintenance: updateNodes.sh <resource-group> <cluster-name> <nodepool-name> [apply]
```

File: README [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+Sample Output:
+
+
+```
+docker run -it appuio/aks-maintenance updateNodes.sh xxxx xxxx workers
```
Comment (anothertobi) [inline README]: Same here with the domain:
```suggestion
docker run -it docker.io/appuio/aks-maintenance updateNodes.sh xxxx xxxx workers
```

**Review by anothertobi — CHANGES_REQUESTED** [review]
Please also add a newline to the end of each file.

**Comment by anothertobi** [comment]
Added the needed environment secrets for docker.io and quay.io to the repository

File: 2.9.1/Dockerfile (hunk 2 above)
Comment (splattner) [reply 2.9.1/Dockerfile]: :joy: :+1:

**Comment by splattner** [comment]
> Added the needed environment secrets for docker.io and quay.io to the repository

thx!

File: LICENCE (hunk 1 above)
Comment (splattner) [reply LICENCE]: copy/paste from one of your other repos :see_no_evil:

**Review by anothertobi — APPROVED** [review]

**Pull request merged**